raphtory-graphql/src/data.rs

```diff
@@ -85,94 +107,825 @@ impl Data {
                 loader(path)
             })
     }
+}
 
-    pub fn load_from_file(path: &str) -> HashMap<String, IndexedGraph<MaterializedGraph>> {
-        fn get_graph_name(path: &Path, graph: &MaterializedGraph) -> String {
-            graph
-                .properties()
-                .get("name")
-                .into_str()
-                .map(|v| v.to_string())
-                .unwrap_or_else(|| path.file_name().unwrap().to_str().unwrap().to_owned())
-        }
+fn copy_dir_recursive(source_dir: &Path, target_dir: &Path) -> Result<()> {
```

Comment:
Could this not accidentally copy your entire drive? I think we want to limit load from dir to only be .raph files within the specified dir